draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1434,8 +1434,8 @@ The IANA registry for this section is {{IANA-RTP-CHE}}.
 | urn:ietf:params:rtp-hdrext:csrc-audio-level | Mixer-to-client audio level indicators | {{?RFC6465}} | no |
 | urn:3gpp:video-orientation:6 | Higher granularity (6-bit) coordination of video orientation (CVO) feature, see clause 6.2.3 | {{3GPP-TS-26.114}} | probably not(?) |
 | urn:3gpp:video-orientation | Coordination of video orientation (CVO) feature, see clause 6.2.3 | {{3GPP-TS-26.114}} | probably not(?) |
-| urn:3gpp:roi-sent | Signalling of the arbitrary region-of-interest (ROI) information for the sent video, see clause 6.2.3.4 | {{3GPP-TS-26.114}} | probably not(?) |
-| urn:3gpp:predefined-roi-sent | Signalling of the predefined region-of-interest (ROI) information for the sent video, see clause 6.2.3.4 | {{3GPP-TS-26.114}} | probably not(?) |
+| urn:3gpp:roi-sent | Signaling of the arbitrary region-of-interest (ROI) information for the sent video, see clause 6.2.3.4 | {{3GPP-TS-26.114}} | probably not(?) |
+| urn:3gpp:predefined-roi-sent | Signaling of the predefined region-of-interest (ROI) information for the sent video, see clause 6.2.3.4 | {{3GPP-TS-26.114}} | probably not(?) |
```

Comment:
For these 3GPP URNs, we might or might not want to change the spellings, because I think these match [entries in 3GPP documents](https://www.3gpp.org/3gpp-groups/core-network-terminals-ct/ct-wg1/uniform-resource-identifier-uri-list) that spell it as "signalling". I don't know if that really matters, and if it does, the RFC Editor is going to be the arbiter of taste anyway, but I should ask while I'm reviewing the PR, right?

Reply:
Good point, I reverted these and also the *Generic negative acknowledgement* above.

Reply:
Thanks! It really does take teamwork ...